docs/lbann.rst

```diff
@@ -0,0 +1,72 @@
+********************
```

Comment:
I find this entire description very unclear. I would start be defining first what you mean by a model, since you use that to define what a trainer is.

Beyond this, it is a strange mixture of very high level and very low level details. It doesn't really seem to define much semantics.

Reply:
Ah this was the one content change I made since the page should have a title... Call it optimism. I didn't actually read the text; I was just trying to get it to render.

Reply:
In the future I will try to revise this more.